### ISSUE TITLE
QDate with range and custom locale has no value for $pluralDay

### ISSUE THREAD

**Issue by rabalyn** [quasarframework/quasar#12396]
### What happened?

QDate with range option and custom locale shows "undefined" for `pluralDay`

### What did you expect to happen?

`pluralDay` should be mentioned in the QDate docs and should use the quasar default locale value of `pluralDay` if missing in the custom `locale` object

### Reproduction URL

https://jsfiddle.net/x2q0dLjv/2/

### How to reproduce?

Go to jsfiddle

Rendered Calendar shows the range of "2 undefined"

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Components (quasar)

### Platforms/Browsers

Firefox, Chrome

### Quasar info output

```shell
Operating System - Linux(5.10.60.1-microsoft-standard-WSL2) - linux/x64
NodeJs - 16.13.1

Global packages
  NPM - 8.3.0
  yarn - Not installed
  @quasar/cli - 1.2.2
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.5.3 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app - 3.3.2 -- Quasar Framework local CLI
  @quasar/extras - 1.12.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - 1.1.0 -- Official ESLint plugin for Quasar
  vue - 3.2.30 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.12
  vuex - 4.0.2 -- state management for Vue.js
  electron - 15.3.6 -- Build cross platform desktop apps with JavaScript, HTML, and CSS
  electron-packager - 15.4.0 -- Customize and package your Electron app with OS-specific bundles (.app, .exe, etc.) via JS or CLI
  electron-builder - Not installed
  @babel/core - 7.17.0 -- Babel compiler core.
  webpack - 5.68.0 -- Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.
  webpack-dev-server - 4.7.3 -- Serves a webpack app. Updates the browser on changes.
  workbox-webpack-plugin - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  typescript - 4.5.5 -- TypeScript is a language for application scale JavaScript development
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - lokka
  eth0 - 172.19.44.124
  flannel.1 - 10.42.0.0
  cni0 - 10.42.0.1
  br-ab96f6055d80 - 172.18.0.1
```

**Comment by rabalyn** [comment]
Thanks for immediatly fixing the fallback translation @MilosPaunovic!

Any chances to add the `pluralDay` config property to the [QDate locale docs](https://github.com/quasarframework/quasar/blob/dev/ui/src/components/date/use-datetime.json#L17)?

**Comment by MilosPaunovic** [comment]
Absolutely! Thanks for pointing that out; it's added to existing PR.

**Comment by rstoenescu** [comment]
Went with a better route: merge Quasar Language Pack with the supplied locale. This way it improves usability as you need to supply only what you actually need. Updated docs also.

Will be available in Quasar v2.5.5.
